rails/app/controllers/admin/places_controller.rb

```diff
@@ -52,9 +52,17 @@ def import_csv
       else
         filepath = params[:file].read
         errors = Place.import_csv(filepath, current_community)
-        errors.empty? ? flash[:notice] = "Places were imported successfully!" : flash[:error] = errors
+        errors.empty? ? flash[:notice] = "Places were imported successfully! View them #{view_context.link_to 'here', admin_places_path}." : flash[:error] = errors
         redirect_back(fallback_location: root_path)
       end
     end
+
+    def import_page
+      render "import_page"
```

Comment:
@Aliciawyse are those renders necessary? doesn't rails try to render a page with matching name by default?